src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/Patch_V1Compat.txt

```diff
@@ -41,3 +44,238 @@ Errors: Error 13-143: The value passed to the 'Patch' function cannot be changed
     {Properties:{Moon:{Name:"Phobos"}}},
     {Planet:"Jupter"})
 Errors: Error 13-251: The value passed to the 'Patch' function cannot be changed.
+
+>> Patch(t1, r1, r_empty);
+   t1
+Table({Field1:1,Field2:"earth",Field3:DateTime(2022,1,1,0,0,0,0),Field4:true})
+
+>> Patch(t1, r1, {Field2:"mars"});
```

Comment:
>Patch(t1, r1, {Field2:"mars"}); [](http://example.com/codeflow?start=3&length=31)

Here we updated the table `t1`, so that the record which used to match `r1` doesn't match it anymore (we shouldn't be updating r1 when we patch the table). The subsequent Patch calls should not be updating the same record.

Reply:
Each of these test cases dont persist. Each test case is a standalone one.

Reply:
Ah, you're right, I was mixing it up with the tests that run under MutationScripts